src/query/storages/fuse/fuse/src/operations/revert.rs

```diff
@@ -54,8 +54,9 @@ impl FuseTable {
 
         // 4. let's roll
         let tenant = ctx.get_tenant();
-        let db_name = &navigation_descriptor.database_name;
-        let reply = catalog.update_table_meta(&tenant, db_name, req).await;
+        let reply = catalog
+            .update_table_meta(&tenant, self.table_info.db_type.clone(), req)
+            .await;
```

Comment:
Maybe it'd be better to use the `tenant` in `TableInfo` instead of fetching it from ctx, similar to the way `db_type` does. Maybe next PR?

Reply:
No, this PR should do this.